aioesphomeapi/_frame_helper/plain_text.py

```diff
@@ -10,6 +11,47 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+_int = int
+_bytes = bytes
+
+
+def _varuint_to_bytes(value: _int) -> bytes:
+    """Convert a varuint to bytes."""
+    if value <= 0x7F:
+        return bytes((value,))
+
+    result = []
+    while value:
+        temp = value & 0x7F
+        value >>= 7
+        if value:
+            result.append(temp | 0x80)
+        else:
+            result.append(temp)
+
+    return bytes(result)
+
+
+_cached_varuint_to_bytes = lru_cache(maxsize=1024)(_varuint_to_bytes)
+varuint_to_bytes = _cached_varuint_to_bytes
```

Comment:
We make `varuint_to_bytes` a cython object, but if we did it `_cached_varuint_to_bytes` directly it would not get the decorator